src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -91,7 +103,8 @@ def register_submission(
                 {
                     "informatieobjecttype": "informatieobjecttype_attachment",
                     "organisatie_rsin": "organisatie_rsin",
-                    "vertrouwelijkheidaanduiding": "vertrouwelijkheidaanduiding",
+                    "docVertrouwelijkheidaanduiding": "vertrouwelijkheidaanduiding",
+                    "auteur": "auteur",
```

Comment:
see above

Reply:
The `create_document` function raises a `KeyError` if `doc_vertrouwelijkheidaanduiding` is not set.